pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -32,10 +34,58 @@ def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool,
         self.images = images
         self.exclude_arches = [] if not exclude_arches else exclude_arches.replace(',', ' ').split()
         self.skip_multiarch_payload = skip_multiarch_payload
+        self.triggered_from_gen_assembly = triggered_from_gen_assembly
+        self.build_number = build_number
```

Comment:
"build number" is a jenkins concept. Let's call it `job run` instead and get the job run url from `Runtime.get_job_run_url` in case we move to Tekton in the future.

Reply:
Understood! Updated!